src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -681,4 +711,9 @@ private Integer updateStudyFromConversion(User user, Dataset dataset,
     return studyId;
   }
 
+  @VisibleForTesting
+  public void setDatasetBatchSize(Integer DATASET_BATCH_SIZE) {
```

Comment:
If this method is only used in testing, how about removing the `public` modifier and leveraging the default behavior?